src/main/java/com/thedariusz/warnme/api/MeteoAlertController.java

```diff
@@ -1,27 +1,85 @@
 package com.thedariusz.warnme.api;
 
 import com.thedariusz.warnme.twitter.TweetService;
+import com.thedariusz.warnme.user.User;
+import com.thedariusz.warnme.user.repository.UserDao;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.authentication.AnonymousAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
 
-@RestController
+//@RestController
+@Controller
 @RequestMapping(value = "/alerts")
 public class MeteoAlertController {
```

Comment:
You should not change this @RestController to @Controller and instead simply create new Controller i.e `MeteoAlertViewController` which handles only views and leave this one untouched. Right now you mix view controller with rest controller and we still want to have simple rest controller to trigger some admin actions like fetching tweets